### ISSUE TITLE
SDE code

### ISSUE THREAD

**Issue by dgm2** [gracezhao1997/EEGSDE#3]
Hi, thanks for this code.  
Could you please point me to where the blocks of code where the SDEs and score-functions are implemented? 
Since the code is based on the Equivariant Diffusion Model, which is a DDPM model, what is the change to make it score-based ?
Many thanks!

**Comment by gracezhao1997** [comment]
Yes. More details are available at Algorithm 2 in Equivariant Diffusion for Molecule Generation in 3D.
